config/env.config.js

```diff
@@ -55,5 +55,46 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
+    this.apply = function(compiler) {
+        compiler.plugin('compilation', function(compilation) {
+            compilation.plugin('html-webpack-plugin-before-html-processing', function(htmlPluginData, callback) {
+                let htmlHead = htmlPluginData.html.match('<head>').index;
+                htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + '<head>' +
```

Comment:
Can you verify that <head> exists in the document before appending it. If for some reason the page does not have a <head> your script produces invalid HTML.